docs/user/azure/account.md

```diff
@@ -0,0 +1,94 @@
+# Create an Azure account
+
+You can sign up for an Azure account here: 
+
+[Azure: Create your Azure free account today][user-create]
+
+This is the welcome screen. You can create a free account or you can purchase services.
+
+![Create account](images/account_Create_your_Azure_free_account.png)
+
+Once you've opted to create an account, you must fill out information about yourself. All fields are required. 
+
+![Step 1](images/account_Sign_up_1.png)
+
+You will need to verify your identity once you've filled out your information. You can either bet called or sent an SMS message.
+
+![Step 2](images/account_Sign_up_2.png)
+
+Once your identity has been verified, you must provide a credit card. If you are creating a free account, this will not be charged.
+
+![Step 3](images/account_Sign_up_3.png)
+
+After everything is filled out, you must agree to the terms and conditions. 
+
+![Step 4](images/account_Sign_up_4.png)
+
+Now that everything is filled out, you can go to the Azure portal.
+
+![Created](images/account_Welcome_to_Microsoft_Azure.png)
+
+This is what the Azure portal looks like. 
+
+![Portal](images/account_Microsoft_Azure.png)
+
+
+# Set up Azure CLI credentials
+
+To install the Linux Azure CLI tools, follow [these instructions][install-cli]. To start using the command line tools, enter:
+````console
+$ az login
+````
+This will direct you to login via a browser (which can be on another device). The CLI client will automatically retrieve and store the authz token at the end of the process. You can configure the default subscription using:
+````console
+$ az account set -s NAME_OR_ID
+````
+Collect your account info:
+````console
+$ az account show
+{
+  "environmentName": "AzureCloud",
+  "id": "4bab1c60-32d5-30b3-f68e-27a15b958a90",
+  "isDefault": true,
+  "name": "Example Team",
+  "state": "Enabled",
+  "tenantId": "8047c7e9-c2ad-988d-d54e-ac3f6be6a7ee",
+  "user": {
+    "name": "user@example.com",
+    "type": "user"
+  }
+}
+````
+
+The installer (and the cluster credentials provider) needs a Service Principal for Azure API access. There are two ways to do this:
+
+Option 1: Use the Wizard
+
+First create a service principal (replacing --name with your own in the format $user-$purpose or $team-$purpuse, whichever is more appropriate):
+````console
+$ az ad sp create-for-rbac --role Owner --name team-installer
+Retrying role assignment creation: 1/36
+Retrying role assignment creation: 2/36
+Retrying role assignment creation: 3/36
+{
+  "appId": "e564ac9e-bd46-4c5a-8592-03724a82c3f2",
+  "displayName": "team-installer",
+  "name": "http://team-installer",
+  "password": "111-11-1111",
+  "tenant": "6047c7e9-b2ad-488d-a54e-dc3f6be6a7ee"
+}
+````
+When you run the installer wizard, if the credentials file doesn't exist, you'll be prompted to enter account and service principal info and the credentials file will be generated.
```

Comment:
also we should provide them 2 options.

1) using the wizard..

show them the openshift-install create cluster and what values from the above must be entered when..

2) create your own cred file and provide that to the installer using `AZURE_AUTH_LOCATION` env.

Reply:
https://github.com/openshift/installer/blob/master/docs/user/azure/install.md already shows running openshift-install create cluster without the credentials file existing. Should I just create a link to that from this page?

Reply:
I went ahead and adding a link to the install page.